packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -380,6 +381,7 @@ class MenuContent extends React.Component {
               totalItems,
               itemIndex,
               intl: this.props.intl,
+              'aria-describedby': !MenuUtils.isMac() && !this.props.index && this.props.showHeader && index === 0 ? menuTopHeaderId : undefined,
```

Comment:
why do we need aria-described by only for Mac ?

Reply:
This was added (for JAWS) to fix the issue where the header was not announced when main-menu is opened. In case of VO, it does get announced but for jaws, when the focus is on first item, it fails to provide header context.
 Hence have added this for only main-level menu and for the first focusable item only.